app/src/androidTest/java/org/simple/clinic/storage/migrations/BaseDatabaseMigrationTest.kt

```diff
@@ -0,0 +1,37 @@
+package org.simple.clinic.storage.migrations
+
+import androidx.annotation.CallSuper
+import androidx.room.migration.Migration
+import androidx.sqlite.db.SupportSQLiteDatabase
+import org.junit.Before
+import org.junit.Rule
+import org.junit.rules.ExpectedException
+import org.simple.clinic.MigrationTestHelperWithForeignKeyConstraints
+import org.simple.clinic.TestClinicApp
+import javax.inject.Inject
+
+abstract class BaseDatabaseMigrationTest(
```

Comment:
`DatabaseMigrationAndroidTest` has been growing massively in size and getting slower and slower to work with over time. So I introduced this base class so we can have one test class per migration.

This base class provides two properties, `before` and `after`, which will give access to the databases before and after migration.